### PR TITLE
Improve TOTP UX

### PR DESCRIPTION
**Pull request by VivekVinushanth**
### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx [hunk 1]
```diff
@@ -261,7 +261,20 @@ export const TOTPAuthenticator: React.FunctionComponent<TOTPProps> = (
...
         return (
             <Segment basic >
-                    <h5 className=" text-center" > { t(translateKey + "modals.scan.heading") }</h5>
+                { !isTOTPConfigured
+                    ?
+                    <Grid>
+                        <Grid.Row columns={ 1 } centered={ true }>
+                            <Grid.Column mobile={ 12 } tablet={ 12 } computer={ 12 } >
+                                <Message info>
+                                        { "QR code has been successfully added to your profile!" }
```
Comment (thivi) [inline apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx:270]: Shall we localize this?

File: apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx [hunk 2]
```diff
@@ -677,7 +693,7 @@ export const TOTPAuthenticator: React.FunctionComponent<TOTPProps> = (
                                                 />
                                             )
                                         }
-                                        content={ t(translateKey + "hint") }
+                                        content={ "Add new QR code" }
```
Comment (thivi) [inline apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx:696]: Shall we localize this?

**Review by thivi — APPROVED** [review]

File: apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx (hunk 1 above)
Comment (VivekVinushanth) [reply apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx]: Fixed with b5e82899c9c6b76d2ba7e0e0e5e41d10436e5bc0

File: apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx (hunk 2 above)
Comment (VivekVinushanth) [reply apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx]: Fixed with b5e82899c9c6b76d2ba7e0e0e5e41d10436e5bc0

**Review by DinikaSen — APPROVED** [review]

**Pull request merged**